### PR TITLE
Removed Global Styles

### PR DESCRIPTION
**Pull request by AlexGilleran**
Right now terria requires a bunch of global styles that hit individual element names, which makes it hard to integrate into other UIs on a component-by-component basis (you've pretty much got to use an iframe), and means we can't generate a decent styleguide.

This removes those styles, making terriajs more modular, and is the first step to allowing it to be used as a toolkit where you pick certain components rather than being forced to use the StandardUserInterface.

### CODE REVIEW

File: lib/ReactViews/Preview/MappablePreview.jsx [hunk 1]
```diff
@@ -38,7 +38,7 @@ const MappablePreview = React.createClass({
             <div className={Styles.root}>
                 <If condition={catalogItem.isMappable}>
                     <DataPreviewMap terria={this.props.terria} previewedCatalogItem={this.props.previewed}
-                                    showMap={!this.props.viewState.explorerPanelAnimating} />
+                                    showMap={!this.props.viewState.explorerPanelAnimating || this.props.viewState.useSmallScreenInterface} />
```
Comment (AlexGilleran) [inline lib/ReactViews/Preview/MappablePreview.jsx:41]: derp

**Comment by chloeleichen** [comment]
All look good to me, I think it can be merged once https://github.com/TerriaJS/TerriaMap/pull/61 is merged into NM

**Comment by AlexGilleran** [comment]
Can we merge that then @chloeleichen ? It's super small :)

**Comment by chloeleichen** [comment]
Does the PR on TerriaMap automatically get merged into NM when merged? or do we need to create seperate PR? @AlexGilleran

**Comment by AlexGilleran** [comment]
Nah it doesn't get automatically merged, but I don't think it needs a PR either.

**Comment by AlexGilleran** [comment]
That TerriaMap one is merged into NM now :)

**Pull request merged**